src/domains/user/service.ts

```diff
@@ -0,0 +1,5 @@
+import { api } from "@/services/api";
+
+export async function getMe() {
+    return await api.get("me");
+}
```

Comment:
Usually you can omit both async and await in similar functions. It'll still work the same (as `api.post` returns `Promise`)

```suggestion
export function getMe() {
    return api.get("me");
}
```